jest.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
       'identity-obj-proxy',
     '^@assets(.*)&': '<rootDir>/src/assets$1',
     "^@components(.*)$": "<rootDir>/src/components$1",
```

Comment:
```suggestion
    '^@components(.*)$": "<rootDir>/src/components$1',
```
왜 해당 라인만 double quote 일까요? 수정이 필요할듯합니다.

Reply:
그보다 우선적으로 해당 branch에 맞지 않는 파일로 보입니다. @dididy @BKJang

Reply:
이 부분은 alias PR에 반영하도록 하겠습니다.